components/molecules/BenefitCard.js

```diff
@@ -1,23 +1,94 @@
+import { EntitlementBox } from "../atoms/EntitlementBox";
+import {
+  ChartPieIcon,
+  ClockIcon,
+  CurrencyDollarIcon,
+} from "@heroicons/react/solid";
+import { Share } from "../atoms/Share";
 
 export function BenefitCard(props) {
   const { t } = useTranslation("common");
   return (
     <div
-      className={`md:shadow-md h-auto min-h-96 w-full rounded-md border pl-3 pr-3 
+      className={` flex flex-col  h-50 w-full hover:shadow-cards border border-b-4 border-green-600 pl-3 pr-3
       ${!props.eligibility ? "bg-gray-300" : ""}`}
     >
-      <div className="py-2">
-        <small className="text-gray-500 uppercase">{props.type}</small>
+      <div className="flex justify-between pt-2 ">
         <a href="#catalog">
           <h2 className="text-h3">{props.title}</h2>
         </a>
+        {/* share link href and funtionality still to be determined */}
+        <Share href="#" text={t("share")}></Share>
       </div>
-      <p className="pb-4">{props.description}</p>
-      <div className="pb-5 pt-1 flex justify-end justify-items-end">
+      <div className="my-2 pb-1">
+        <small className="bg-gray-800 text-white uppercase px-1 py-1 rounded-sm">
+          {props.type}
+        </small>
+      </div>
+      {/* Entitlement info */}
+      <EntitlementBox>
+        <EntitlementInfo
+          icon={<CurrencyDollarIcon className=" text-green-800" />}
+          bgColor="bg-green-100"
+          textColor="text-green-900"
+          title={t("howMuchTitle")}
+          body={t("howMuchBody")}
+          dataCy="how-much"
+        />
+        <EntitlementInfo
+          icon={<ChartPieIcon className=" text-blue-800" />}
+          bgColor="bg-blue-100"
+          textColor="text-blue-900"
+          title={t("howLongTitle")}
+          body={t("howLongBody")}
+          dataCy="how-long"
+        />
+        <EntitlementInfo
+          icon={<ClockIcon className=" text-red-800" />}
+          bgColor="bg-red-100"
+          textColor="text-red-900"
+          title={t("howSoonTitle")}
+          body={t("howSoonBody")}
+          dataCy="how-soon"
+        />
+      </EntitlementBox>
+      {/* entitlement info end */}
+      {/* benefit description start */}
+      <div className="flex-1 p-4">
+        <p className="text-xl font-extrabold py-3">{t("overview")}</p>
+        {props.description ? (
+          <span>
+            <p className="prose md:max-w-none">{props.description}</p>
+          </span>
+        ) : (
+          "Content coming soon!"
+        )}
+
+        {/* don't show the criteria section if there isn't anything in strapi for EligibilityCriteria */}
+        {props.eligibilityCriteria ? (
+          <div>
+            <p className="text-xl font-body font-extrabold py-3">
+              {t("amIEligible")}
+            </p>
+            <span className="text-sm font-semibold pl-1">{t("youMayBe")}</span>
+            <p className="prose md:max-w-none mt-1">
```

Comment:
This p tag is not necessary, the ReactMarkdown provides a div. You can take the className from the p tag and put it on the ReactMarkdown element to then remove the p tag.

Reply:
ok